README.md

```diff
@@ -186,6 +186,8 @@ There are three artifacts with extra support for Android:
 * `autodispose-android-archcomponents` has a `AndroidLifecycleScopeProvider` for use with `LifecycleOwner` and `Lifecycle` implementations.
 * `autodispose-android-archcomponents-test` has a `TestLifecycleOwner` for use in testing.
 
+Kotlin extension artifacts are a [WIP for the next release](https://github.com/uber/AutoDispose/issues/340).
```

Comment:
Generally shouldn't mark WIP stuff in the README, let's remove this for now. README should just reflect current master

Reply:
Makes sense d12ba8c3c8f7a1e622cbe58fb887827a8134b0af